www/js/diary/detail.js

```diff
@@ -32,4 +32,44 @@ angular.module('emission.main.diary.detail',['ui-leaflet',
   $scope.tripgj = DiaryHelper.directiveForTrip($scope.trip);
   
   console.log("trip.start_place = " + JSON.stringify($scope.trip.start_place));
+
+  var data  = [];
+  var totalDistance = 0;
+  for (var s in $scope.tripgj.sections) {
+    for (var p in $scope.tripgj.sections[s].properties.distances) {
+      totalDistance += $scope.tripgj.sections[s].properties.distances[p];
```

Comment:
I thought that my feedback was to have speed versus time. You get the time from the points, which are ~ 30 secs apart.